endpoints/globus/endpoint.go

```diff
@@ -574,25 +573,23 @@ func (ep *Endpoint) Cancel(id uuid.UUID) error {
 	// 3. "TaskComplete", indicating that the task is complete and not able to
 	//    be canceled.
 	//
-	// To avoid a 10-second wait, we simply issue a request asynchronously and
-	// settle for a "best-effort" execution, which (it seems to me) is just a less
-	// elaborate framing of what Globus gives us.
-
-	errChan := make(chan error, 1) // <-- captures immediately issued errors
-	go func() {
-		resource := fmt.Sprintf("task/%s/cancel", id.String())
-		_, err := ep.post(resource, nil) // can take up to 10 ѕeconds!
-		if err != nil {
-			errChan <- err
-			return
+	// We live with the 10-second wait for now, since our polling interval is
+	// large.
+	type CancellationResponse struct {
+		Code      string `json:"code"` // should be "Canceled"
+		Message   string `json:"message"`
+		RequestId string `json:"request_id"`
+		Resource  string `json:"resource"`
+	}
+	resource := fmt.Sprintf("task/%s/cancel", id.String())
+	_, err := ep.post(resource, nil) // can take up to 10 ѕeconds!
```

Comment:
:meow_tableflip:

Reply:
I know. I can dispatch this to a persistent `stupid_globus_cancellation_queue` goroutine that just lets it finish at its leisure if we perceive this to be an issue,